bee-node/src/tools/benchmark_cpu.rs

```diff
@@ -97,14 +97,16 @@ pub fn exec(tool: &BenchmarkCPUTool) {
 }
 
 fn cpu_benchmark_worker(_pow_digest: &[u8], start_nonce: u64, cancel: MinerCancel, counter: Arc<AtomicU64>) {
-    let mut pow_digest = TritBuf::<T1B1Buf>::new();
+    let HASH_LENGTH = 243;
+    
+    let mut pow_digest = TritBuf::<T1B1Buf>::with_capacity(HASH_LENGTH);
     b1t6::encode::<T1B1Buf>(&_pow_digest)
         .iter()
         .for_each(|t| pow_digest.push(t));
 
     let mut nonce = start_nonce;
-    let mut curlp = CurlP::new(CurlPRounds::Rounds81);
-    let mut buffers = TritBuf::<T1B1Buf>::new();
+    let mut curlp = CurlP81::new();
+    let mut buffers = TritBuf::<T1B1Buf>::with_capacity(HASH_LENGTH);
```

Comment:
Is it really `HASH_LENGTH` ? Seems to me that the final size of `buffers` is `BATCH_SIZE * (pow_digest.len() + nonce_trits.len())` ?

Reply:
It looks like that's what we use in `bee-pow`: https://github.com/iotaledger/bee/blob/c08f1bac7170fe5cb650ddc347ac4d483bb9036a/bee-pow/src/score.rs#L29